nixos/lib/systemd-unit-options.nix

```diff
@@ -94,131 +94,133 @@ in rec {
 
   };
 
-  commonUnitOptions = { options = (sharedOptions // {
+  commonUnitOptions = {
+    options = (sharedOptions // {
```

Comment:
Preexisting problem: redundant parentheses and better solved with module composition. Can scope this out.

Reply:
I was trying that and: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/boot/networkd.nix#L1737